extra/cloudformation/default-grab-resources-template.yaml

```diff
@@ -0,0 +1,228 @@
+      MapName: location.aws.com.demo.maps.Esri.DarkGrayCanvas
+
+  AmazonLocationDemoMapsEsriImagery:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: RasterEsriImagery
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.Esri.Imagery
+
+  AmazonLocationDemoMapsEsriLightGrayCanvas:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorEsriLightGrayCanvas
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.Esri.LightGrayCanvas    
+
+  AmazonLocationDemoMapsEsriNavigation:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorEsriNavigation
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.Esri.Navigation
+
+  AmazonLocationDemoMapsHere:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorHereExplore
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.HERE.Explore
+
+  AmazonLocationDemoMapsHereContrast:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorHereContrast
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.HERE.Contrast     
+
+  AmazonLocationDemoMapsHereExploreTruck:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorHereExploreTruck
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.HERE.ExploreTruck
+
+  AmazonLocationDemoMapsHereHybrid:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: HybridHereExploreSatellite
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.HERE.Hybrid
+
+  AmazonLocationDemoMapsHereImagery:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: RasterHereExploreSatellite
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.HERE.Imagery
+
+  AmazonLocationDemoMapsGrabLight:
+    Type: AWS::Location::Map
+    Properties: 
+      Configuration: 
+        Style: VectorGrabStandardLight
+      Description: You created this resource by running Amazon Location Demo's CloudFormation template.
+      MapName: location.aws.com.demo.maps.Grab.StandardLight
+
+  AmazonLocationDemoMapsGrabLight:
```

Comment:
2 resources with the same name. CFN would through validation error

Reply:
resolved and updated